core/src/main/java/nl/nn/adapterframework/extensions/mqtt/MqttListener.java

```diff
@@ -171,4 +171,9 @@ public Message extractMessage(MqttMessage rawMessage, Map<String, Object> contex
 	@Override
 	public void afterMessageProcessed(PipeLineResult processResult, Object rawMessageOrWrapper, Map<String, Object> context) throws ListenerException {
 	}
+
+	@Override
+	public ClassLoader getConfigurationClassLoader() {
+		return super.getClassLoader();
+	}
```

Comment:
Shouldn't you put this in the MqttFacade? It already has a configure, just no `IConfigurable` interface